scripts/tasks/screener.ts

```diff
@@ -12,33 +11,17 @@ import { getStorybook } from '@storybook/react';
 export async function screener() {
   const screenerConfigPath = path.resolve(process.cwd(), './screener.config.js');
   const screenerConfig: ScreenerRunnerConfig = require(screenerConfigPath);
-  console.log('screener config for run');
+  console.log('screener config for run:');
   console.log(JSON.stringify(screenerConfig, null, 2));
 
-  const packageInfos = getAllPackageInfo();
-  const packagePath = path.relative(findGitRoot(), process.cwd());
-  const affectedPackageInfo = Object.values(packageInfos).find(x => x.packagePath === packagePath);
-  let affectedPackages = new Set<string>();
-  const isPrBuild = process.env.BUILD_SOURCEBRANCH && process.env.BUILD_SOURCEBRANCH.includes('refs/pull');
-
-  if (isPrBuild) {
-    affectedPackages = getAffectedPackages();
-  } else {
-    // master CI build,
-    const previousMasterCommit = getNthCommit();
-    affectedPackages = getAffectedPackages(previousMasterCommit);
-  }
-
-  debugAffectedGraph(affectedPackages);
-
   try {
-    if (!affectedPackages.has(affectedPackageInfo.packageJson.name)) {
-      await cancelScreenerRun(screenerConfig, 'skipped');
-    } else {
+    console.log(process.env.SKIPSCREENERBUILD);
+    if (process.env.SKIPSCREENERBUILD !== 'true') {
```

Comment:
can we add this to `environments` in screener.runner.ts` so we keep track of all env variables related to screener in one place ?